src/main/java/com/airlines/dao/impl/CrewMemberDaoImpl.java

```diff
@@ -40,7 +40,7 @@ public void save(CrewMember crewMember) {
             ResultSet generatedKeys = insertStatement.getGeneratedKeys();
             if (generatedKeys.next()) {
                 long id = generatedKeys.getLong(1);
-                crewMember.setId(id);
+                CrewMember.builder().withId(id);
```

Comment:
This line doesn't have any sense I guess.

Previous flow with crewMember.setId(id);

we have some object, CrewMember with all fields but without id. We save it to database. And set id for this entity.
So:
```
CrewMember draft = new CraftMember(...) //without id
save(draft);  //saving draft to database and setting id to this object
...
Long id = draft.getId(); // now we have our id a
```

But with builder it became more complicated. Our  entity is immutable. It couldn't be updated with id.
And this line:
` CrewMember.builder().withId(id);`
Means that new CrewMember object is created, with only one field - id.
All other fields are missing , you could easily check it by making 
public static void main (String...args) method
 and code there:
CrewMember crewMember = CrewMember.builder().withId(id);
System.out.println(cerwMember);
only id is set.

and moreover, 
line
` CrewMember.builder().withId(id);`
do nothing, because this line returns builder, and you did nothing with it.

It's like:

`list.get(0);`
line, you retrieve element, but you do nothing with this object, Usually, idea will mark it by grey or yellow color and message:
"return value is ignored"

I don't know actually, what it's better to do...I guess we have two options:
1) just drop 
```
        if (generatedKeys.next()) {
                long id = generatedKeys.getLong(1);
                CrewMember.builder().withId(id);
            }
```
because it do nothing

2)  change method signature to:
`public CrewMember save(CrewMember crewMember) {`
then
```
        if (generatedKeys.next()) {
                long id = generatedKeys.getLong(1);
                return copyAttributesAndSetId(crewMember, id);
            }
```
 private CrewMember copyAttributesAndSetId(CrewMember crewMember, Long id) {
   return CrewMember.builder()
            .withId(id)
           .withFirstName(crewMember.getFirstName)
           ...
}
and just copy all attributes.

Do option #2